Внедрён абстрактный класс SimpleController

argoneen1 opened the pull request:
Предыдущий мержанул случайно

argoneen1 committed:
Внедрён абстрактный класс SimpleController

eugene-lenkevich reviewed (changes requested):
Привет, Александр , спасибо за PR

В этом ПР, да, только часть изменений, но в другой части вроде как криминала какого то нет, не критично :)

Вообще, вижу ты оригинально подошел к заданию, абстрактные контроллеры и тд и тп, это круто, но тут лучше не увлекаться, главное не переусложнить :)

Пару моментиков отметил, посмотри комментарии ниже, пожалуйста

argoneen1 committed:
Интерфейс IntId отрефакторен в BaseMode.
SimpleController.getElementName и SimpleController.getExistException сделаны абстрактными

eugene-lenkevich reviewed (approved):
Все супер, спасибо!
